src/components/customer/customerApi.js

```diff
@@ -9,10 +9,8 @@ customerRouter.use("/", (req, res, next) => {
   next();
 });
 
-customerRouter.get(
-  "/company-customers",
-  verifyToken.verifyToken,
-  getCompanyCustomers
```

Comment:
This api is requested by mobile and front-end teams. I'm not sure whether they use it or not. Thus, removing is quite risky

Reply:
Seem like it is just a rename. As I see, Khoi updated it on api docs.